spec/models/item_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  it "is valid with name, description, price" do
+    item = Item.create(name: 'Canon 7d',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.